tests/test_skills/test_error/test_error_skill_handlers.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the tests for aea.aea.py."""
+
+import time
+from pathlib import Path
+from threading import Thread
+
+from aea.aea import AEA
+from aea.connections.local.connection import LocalNode, OEFLocalConnection
+from aea.crypto.wallet import Wallet
+from aea.mail.base import MailBox, Envelope
+from aea.protocols.default.message import DefaultMessage
+from aea.protocols.default.serialization import DefaultSerializer
+from aea.protocols.fipa.message import FIPAMessage
+from aea.protocols.fipa.serialization import FIPASerializer
+from aea.protocols.oef.message import OEFMessage
+from aea.skills.base import SkillContext
+from tests.conftest import CUR_PATH
+
+from aea.skills.error.handlers import ErrorHandler
+
+
+class TestSkillError:
+    """Test the skill: Error."""
+
+    @classmethod
+    def setup_class(cls):
+        """Test the initialisation of the AEA."""
+        cls.node = LocalNode()
+        cls.wallet = Wallet({'default': None})
+        cls.mailbox1 = MailBox(OEFLocalConnection(cls.wallet.public_keys['default'], cls.node))
+        cls.my_aea = AEA("Agent0", cls.mailbox1, cls.wallet, directory=str(Path(CUR_PATH, "data/dummy_aea")))
+
+        cls.skill_context = SkillContext(cls.my_aea.context)
+        cls.my_error_handler = ErrorHandler(skill_context=cls.skill_context)
+
+    def test_error_handler_setup(self):
+        """Test the setup."""
+        self.my_error_handler.setup()
```

Comment:
These aren't really testing anything. You are just calling these methods on the error handler.

Reply:
yes it is in-order to call the `pass` on the handler. Didn't want to add` # pragma: no cover.`